kinto/plugins/history/__init__.py

```diff
@@ -0,0 +1,17 @@
+from kinto.core.events import ResourceChanged
+
+from .listener import on_resource_changed
+
+
+def includeme(config):
+    config.add_api_capability('history',
```

Comment:
No, it should be part of the MVP, if we use a Listener this will be already covered.